kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -2160,8 +2160,11 @@ object KyuubiConf {
   val OPERATION_LANGUAGE: ConfigEntry[String] =
     buildConf("kyuubi.operation.language")
       .doc("Choose a programing language for the following inputs" +
-        " <ul><li>SQL: (Default) Run all following statements as SQL queries.</li>" +
-        " <li>SCALA: Run all following input a scala codes</li></ul>")
+        "<ul>" +
+        " <li>SQL: (Default) Run all following statements as SQL queries.</li>" +
+        " <li>SCALA: Run all following input as scala codes</li>" +
+        " <li>PYTHON: (Experimental) Run all following input as PySpark codes</li>" +
```

Comment:
which is better `as PySpark cods`, `as Python codes` or `as Python codes with Spark engine`?

Reply:
+1 for `as Python codes with Spark engine`

Reply:
Good, updated.